onadata/libs/tests/utils/test_email.py

```diff
@@ -105,3 +105,8 @@ def test_get_verification_email_data_with_verification_and_redirect_urls(
         self.assertIn(
             encoded_url.replace('&', '&amp;'), email_data.get('message_txt')
         )
+
+    def test_email_data_does_not_contain_newline_chars(self):
+        email_data = self._get_email_data(include_redirect_url=True)
+        self.assertNotIn('/n', email_data.get('message_txt'))
+        self.assertNotIn('/n', email_data.get('subject'))
```

Comment:
Should this be the newline chars? `\n` hence

```python
self.assertNotIn('\n', email_data.get('subject'))
```
Should it also only apply to the email subject? The message body should be able to support newline characters.

Reply:
I am wondering, would it be possible to inject specific subject that has newline and then test that the resulting actually does not have the injected newline? Hence:
```python
subject = "\nHello\nWorld!\n"
...
email_data = self._get_email_data(include_redirect_url=True)
self.assertEqual(email_data.get('subject'), "Hello World!")
```